yt_dlp/extractor/ctvnews.py

```diff
@@ -65,4 +65,9 @@ def ninecninemedia_url_result(clip_id):
             })
             entries = [ninecninemedia_url_result(clip_id) for clip_id in orderedSet(
                 re.findall(r'clip\.id\s*=\s*(\d+);', webpage))]
+            if not entries:
+                webpage = self._download_webpage(url, page_id)
+                if 'getAuthStates("' in webpage:
+                    entries = [ninecninemedia_url_result(clip_id) for clip_id in
+                               re.search(r'getAuthStates\("([\d+,]+)"', webpage).groups()[0].split(',')]
```

Comment:
`self._search_regex`

otherwise can merge

Reply:
re is already being imported and used, is there any advantage here to use search_regex especially when this code won't even be ran if getAuthStates is not even in the webpage?

Reply:
the same reason it is used everywhere else. Convention and better error message
```suggestion
                               self._search_regex(r'getAuthStates\("([\d+,]+)"', webpage, 'clip ids').split(',')]
```